contrib/epee/include/storages/http_abstract_invoke.h

```diff
@@ -44,8 +47,11 @@ namespace epee
       if(!serialization::store_t_to_json(out_struct, req_param))
         return false;
 
+      http::fields_list additional_params;
+      additional_params.push_back(std::make_pair("Content-Type","application/json; charset=utf-8"));
```

Comment:
Do you know why this worked before ? It seems OK, but odd it might need changing, so maybe knowing why will give a clue to a bug.